data/mods/gen7/bss-factory-sets.json

```diff
@@ -160,6 +160,7 @@
 			"species": "Aron",
 			"item": "Berry Juice",
 			"ability": "Sturdy",
+			"evs": {},
```

Comment:
Why? This is intentional so you don’t have empty objects

Reply:
I think it's better to have one empty object for one set than to allow sets that don't have EVs by accident.